contrib/fuzz/seed_corpus_from_chain.py

```
@@ -0,0 +1,341 @@
+    if datadir:
+        cmd.append(f"-datadir={datadir}")
+    cmd.extend(args)
+    try:
+        result = subprocess.run(cmd, capture_output=True, text=True, check=True, timeout=30)
+        return result.stdout.strip()
+    except (subprocess.CalledProcessError, FileNotFoundError, subprocess.TimeoutExpired) as e:
+        print(f"WARNING: dash-cli {' '.join(args)} failed: {e}", file=sys.stderr)
+        return None
+
+
+def save_corpus_input(output_dir, target_name, data_hex, label=""):
```

Ruff (0.15.1)

[warning] Unused function argument: `label`

(ARG001)